ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -213,6 +210,15 @@ uplink_ptp:
 # The configuration is intended to be used as multicast underlay for EVPN OISM overlay
 underlay_multicast: < boolean | default -> false >
 
+# Enable Fabric to support EVPN Multicast
+# Requires "underlay_multicast: true" and IGMP snooping enabled globally.
+# For MLAG devices Route Distinguisher must be unique since this feature will create multi-vtep configuration.
+evpn_multicast: <boolean | default -> false >
+
+# Disable IGMP snooping at fabric level.
+# If set, it overrides per vlan settings
```

Comment:
>>If set, it overrides per vlan settings

I would suggest to specify what does it mean "set" here. Because if set to "true", then it will not override per vlan settings. right?

Reply:
I believe that is correct, This knob was only moved location but not created in this PR.
@ClausHolbechArista

Reply:
updated with `If set to false`, please feel free to modify as needed!